### ISSUE TITLE
Accessibility - Multiple markers not accounted for in single data point

### ISSUE THREAD

**Issue by magdalena-gut** [highcharts/highcharts#19756]
#### Actual behaviour
Accessibility fails to consider that one point could have more than one marker.

This issue affects the arearange/dumbbell series when the new option `lowMarker` is enabled; it appears when the accessibility module is added - #14024

The demo is currently working via` feature/14024-dumbbell-customization` branch. Test it by adding or removing the accessibility module. When the accessibility module is added, the "lowMarker" doesn't show up on the chart.
https://jsfiddle.net/BlackLabel/rzveo2L0/

**Comment by magdalena-gut** [comment]
FYI @oysteinmoseng , any chance we can find a workaround for this or get it into the Core Team sprint?

**Comment by oysteinmoseng** [comment]
Internal note: Probably related to forced markers

**Comment by MarkusBarstad** [comment]
Seems to be fixed for me in 11.3.0: https://jsfiddle.net/ro5g26vz/3/
